FirebaseAppCheck/Sources/Core/FIRAppCheck.m

```diff
@@ -171,7 +172,7 @@ - (instancetype)initWithAppName:(NSString *)appName
 + (instancetype)appCheck {
   FIRApp *defaultApp = [FIRApp defaultApp];
   if (!defaultApp) {
-    [NSException raise:kFIRAppCheckErrorDomain
+    [NSException raise:FIRAppCheckErrorDomain
                 format:@"The default FirebaseApp instance must be configured before the default"
                        @"AppCheck instance can be initialized. One way to ensure that is to "
                        @"call `[FIRApp configure];` (`FirebaseApp.configure()` in Swift) in the App"
```

Comment:
Not necessarily for this PR, but we should start being Swift first in error messages.

Reply:
Will audit repo and issue a follow-up PR with Swift first error messages.